docs/src/pages/components/Text/NativeText.mdx

```diff
@@ -0,0 +1,37 @@
+---
+platform: native
+githubPath: bpk-component-text
```

Comment:
Probably need an npm link for this?

Reply:
Surely we don't need to specify the NPM/Cocoapods link for every mobile component as they would all be the same?

Reply:
Yeah George is correct, we use `platform: native` as a way of knowing whether to add the NPM link or not, as they're all under the same package.

When we migrate this for web we'll need to do it differently.

Reply:
With that point then would we need to be putting this on the bottom of every page for iOS and Android components? As they are also in the same package

https://github.com/Skyscanner/backpack-docs/pull/829/files#diff-e01ea861468b00790a09459df815798b9eba92801f5b41d8cf0658ec2c1e2cd8R59

Reply:
I don't really understand the question, but I've made it so that Android MDX pages get a link to JitPack and iOS ones get a link to CocoaPods. The line you've linked to demonstrates that